src/view/com/util/post-embeds/VideoEmbed.web.tsx

```diff
@@ -42,6 +43,16 @@ export function VideoEmbed({embed}: {embed: AppBskyEmbedVideo.View}) {
     return () => observer.disconnect()
   }, [sendPosition, isFullscreen])
 
+  // In case scrolling hasn't started yet, send up the position
+  const isAnyViewActive = currentActiveView !== null
+  useEffect(() => {
+    if (ref.current && !isAnyViewActive) {
+      const rect = ref.current.getBoundingClientRect()
+      const position = rect.y + rect.height / 2
+      sendPosition(position)
+    }
+  }, [isAnyViewActive, sendPosition])
+
```

Comment:
Can't see a reason it needs to be in the child component. Cleaner if we move it up